config.yml

```diff
@@ -32,6 +73,78 @@ domains:
       - name: Remove device
         description: Unregister a device
         query: mutation.removeDevice
+      - name: IPMI Command
+        description: Run an IPMI command against a device (deprecated)
+        query: mutation.ipmiCommand
```

Comment:
Let's remove this totally dead endpoint from the code rather than documenting it here!

Reply:
https://github.com/nbycomp/graphql-hello-world/pull/1023